### PR TITLE
Created "Added to Contacts" notification

### PR DESCRIPTION
**Pull request by wadeking98**
Signed-off-by: wadeking98 <wkingnumber2@gmail.com>

# Summary of Changes

Added notification to let user know that a connection has been added to their contacts when being offered a credential

# Related Issues

N/A

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: core/App/localization/en/index.ts [hunk 1]
```diff
@@ -150,7 +150,8 @@ const translation = {
   },
   "Connection": {
     "JustAMoment": "Just a moment while we make a secure connection...",
-    "TakingTooLong": "This is taking longer than usual. You can return to home or continue waiting."
+    "TakingTooLong": "This is taking longer than usual. You can return to home or continue waiting.",
+    "ConnectionAdded": "Added to Contacts"
```
Comment (amanji) [inline core/App/localization/en/index.ts:154]: This should be in the credential offer group

File: core/App/screens/Connection.tsx [hunk 2]
```diff
@@ -103,6 +105,10 @@ const Connection: React.FC<ConnectionProps> = ({ navigation, route }) => {
     if (notificationRecord) {
       switch (notificationRecord.type) {
         case 'CredentialRecord':
+          Toast.show({
```
Comment (amanji) [inline core/App/screens/Connection.tsx:108]: Are we changing the spec to do a toast here or is this supposed to be part of the credential offer screen?

Comment (amanji) [reply core/App/screens/Connection.tsx:108]: This is the current wireframe https://xd.adobe.com/view/3db881d4-e963-42ec-a2c4-6b5ca23862a3-68eb/screen/a523d262-2549-4da9-9c2e-35ba1a84d7e8/

Comment (wadeking98) [reply core/App/screens/Connection.tsx:108]: ah, I see. I was missing the wireframe, I'll refactor my changes

Comment (jleach) [reply core/App/screens/Connection.tsx:108]: @wadeking98 If that'll take a bit can you switch this to "Draft". Ty.

**Pull request closed without merging**